guide/getting-started/why-design-for-bitcoin.md

```diff
@@ -37,35 +37,37 @@ Illustration sources
 
 # Why design for bitcoin
 
-At first, Bitcoin may sound like a complex technical innovation. A closer inspection reveals exciting new opportunities for designers to shape user experiences that can make a fundamental difference for people worldwide. So much of our lives is tied to money and financial systems, yet we sometimes have so little control or influence. Bitcoin has the potential to balance this, and it's up to us to make it a reality.
+Bitcoin lets designers seeking the next big challenge build a new kind of money and how we send, spend, save, and store that money. Since Bitcoin’s brand has no owner to set it in stone, you can still have a big impact on Bitcoin even if you aren’t a product designer. And that’s just for starters.
 
-## Be on the cutting edge
+## It’s the internet of money
 
-The Bitcoin white paper was released in 2008. Over a decade later, it is still a young and maturing technology full of potential. Like working early on the internet, there are big and exciting questions to be discussed and design decisions to be made. Designing for bitcoin is a fantastic and rewarding challenge.
+Even though it was released in 2008, Bitcoin is still in its infancy. Working on Bitcoin today is like working on the Internet in its early days—many of the biggest and most exciting problems have yet to be solved or even discovered.
 
-## Permissionless
+## Build what you want
 
-The idea that there are no gatekeepers is baked into the foundation of Bitcoin and its ecosystem. A Bitcoin wallet designed by you is as official as one created by a big corporation.
+Bitcoin is permissionless; this means that there are no individual, corporate, or governmental gatekeepers deciding what’s allowed, so anything that you build has the potential to grow beyond what even the most well-funded can imagine.
 
-## Blaze the trail
+## Blaze trails one pixel at a time
 
-There are still very few dedicated designers working in the Bitcoin ecosystem. As we have seen with the evolution of the internet, it takes time to adopt new technology. If you are curious and excited about new challenges, here's your chance.
+Few open designers currently work full-time on Bitcoin. Most have 9–5 jobs, and like bitcoin’s most open-source developers, contribute when they can. As the ecosystem grows and more investors enter the space, more opportunities to be one of the first full-time bitcoin designers are bound to open up.
```

Comment:
```suggestion
Few open designers currently work full-time on Bitcoin. Most have 9–5 jobs, and like most of Bitcoin’s open-source developers, contribute when they can. As the ecosystem grows, more opportunities to be among the first full-time Bitcoin designers are bound to open up.
```
I'd remove `investors`, as that's not the only funding source for open-source work and the sentence works without it as well.